s3/src/main/scala/geotrellis/store/s3/S3AttributeStore.scala

```diff
@@ -79,18 +78,24 @@ class S3AttributeStore(
         s3objStream.close()
       }
 
-    json.parseJson.convertTo[(LayerId, T)]
+    (for {
```

Comment:
The same question is here; 

```scala
parse(json).flatMap(_.as[(LayerId, T)]).valueOr(throw _)
``` 

I see no reasons to use a `for` comprehension here with a manual match, since we depend on cats anyway.